### DISCUSSION TITLE
ImageSharp Local vs QA Environment

### DISCUSSION THREAD

**Discussion by bbgun75** [SixLabors/ImageSharp.Web#335] [Q&A] [answered]
We've used many versions of ImageSharp over the years and never had any major issues. However this time our local environment is working as expected but when we pushed everything to QA, ImageSharp no longer works. I've tried everything I can think of.  The images aren't resizing, nothing is being written to the ImageSharpCache folder.

Example: https://qa-aem.ascedia.dev/AEM/media/Images/jason-andringa.jpg?width=20&height=60&rmode=crop&ranchor=right&format=webp&quality=85

**Comment by JimBobSquarePants** [comment]
There's not really any information here to go on so let's establish a few details and checks. 

- v3 requires .NET 6. I'm assuming you're using this?
- I'm assuming that the code deployed in your local environment is the same as the deployed code?

I'm not seeing any indication in the response headers that the middleware is intercepting the request at all. 

- Are you using the middleware directly or via a 3rd party e.g. A CMS?
- Are you using any custom providers for the image source or cache?

**Comment by bbgun75** [comment]
If I take my website offline for this project, the site is down except for
the images, they are still displaying. So I'm assuming the images aren't
going through the middleware. All our other sites the images don't display
when the site is offline.

**Reply by JimBobSquarePants** [reply]
I think you need to check the integrity of the deployed application vs local paying particular attention to your registration code. 

Without additional information I'm afraid I cannot be of assistance.

**Reply by JimBobSquarePants** [reply]
Looks like it's working now What was the solution?

**Comment by bbgun75** [comment, marked as answer]
Recreated the project on the server and in IIS. Something was messed up we
couldn't see. Did our normal setup and it's working now. Thanks!

**Reply by JimBobSquarePants** [reply]
Great! I had a suspicion this would be the case.
